doc/source/getting_started/overview.rst

```diff
@@ -6,7 +6,7 @@
 Package overview
 ****************
 
-**pandas** is a `Python <https://www.python.org>`__ package providing fast,
+**pandas** is a `Python <https://www.python.org>`__ package providing fast (`benchmarks <https://pandas.pydata.org/speed/pandas/#/>`__),
```

Comment:
rather put this in the development guide in the asv section :->

Reply:
Yep, that is a much better idea :-)